src/writers/writer.ts

```diff
@@ -16,6 +16,7 @@ export interface IWriteOptions {
   xAlign: string;
   yAlign: string;
   textRotation: number;
+  textShear?: number;
```

Comment:
Oh I see, so it works in tandem with rotation, since they really are separate concepts? Maybe we could rework rotation into "baseDimension" or something -- or, allow any angle for textRotation and internally split it into the two top level properties here?